bin/migratedMyFeedAlert.ts

```diff
@@ -0,0 +1,35 @@
+import '../src/config';
```

Comment:
I think it's better to turn this functionality into a single upsert statement in SQL and run it as part of the migration. This would be the seamless solution for this.